website/src/middlewares/api-v3/static.js

```diff
@@ -0,0 +1,18 @@
+import express from 'express';
+import nconf from 'nconf';
+import path from 'path';
+
+const IS_PROD = nconf.get('IS_PROD');
+const MAX_AGE = IS_PROD ? 31536000000 : 0;
+const PUBLIC_DIR = path.join(__dirname, '/../../../public');
+const BUILD_DIR = path.join(__dirname, '/../../../build');
```

Comment:
yep, probably yes but I don't like the whole concept of how static files are handled here. Ideally before we go live all static files will belong to a single directory so that we can just zip it and upload it somewhere